src/index.js

```diff
@@ -1,14 +1,17 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { BrowserRouter } from 'react-router-dom';
+import { Router } from 'react-router-dom';
```

Comment:
Let's still use `BrowserRouter `

Reply:
The `BrowserRouter` gives a warning when passing the history prop. `Warning: <BrowserRouter> ignores the history prop. To use a custom history, use import { Router } instead of import { BrowserRouter as Router }`